### ISSUE TITLE
Mark active window

### ISSUE THREAD

**Issue by strixBE** [SietsevanderMolen/i3-qubes#1]
Great Patch! Though it would be nice to mark the active window. At the moment, there can be several windows from the same domain with the same color. And therefore it makes it impossible to see, which one you're using. I could implement it myself if I got a hint, where I'd have to put this code (lighten up the active or darken the inactive windows)

**Comment by SietsevanderMolen** [comment]
Thanks for your comment! You're right, this is missing functionality as of now. I didn't implement it because I still want the user to be able to see what domain the active window is and I don't have a problem keeping track of the active window. Lightening or darkening the border seems like a good solution, I like it. You'd probably want to do this in x.c where the big if statement is that sets the right color. The colors right now are just strings, but it should be fairly easy to write a utility function to apply a darkening factor. I just noticed that the original i3 code that sets the focused color is still there, just commented out in my patch. You can most likely just copy and paste that.

Are you going to do this or do you want me to look at it? Either way is fine with me.

**Comment by o-** [comment]
Hey,

thanks for the initial work! Very useful stuff, I am very happy :)

So I have started working on top of your patch at https://github.com/o-/i3/tree/qubes-patch-4.8
This is a fork of i3 with your patch, another patch for the dock swallowing icons on layout changes from debian and my own patch to visually distinguish the active window.

Alternatively the builddir I used is a fork of your repo https://github.com/o-/i3-qubes/tree/qubes-3.1

Feel free to reuse whatever you see fit...

I also played around with newer i3 versions, but got stuck there due to missing libxkbdcommon-x11 in f20 :(

o.
